resources/language/resource.language.es_es/strings.po

```diff
@@ -292,15 +292,15 @@ msgstr "Vista para los exportados"
 
 msgctxt "#30075"
 msgid "Ask to skip intro and recap"
-msgstr "Preguntar si omitir intro y resumen"
+msgstr "Preguntar si omitir la introducción y resumen"
 
 msgctxt "#30076"
 msgid "Skip intro"
-msgstr "Omitir intro"
+msgstr "Saltar introducción"
```

Comment:
I think 'omitir' is more formal than 'saltar' (and 'saltar' may have other meaning: 'jump')

Reply:
![Sin título](https://user-images.githubusercontent.com/24685253/79220492-611f1100-7e54-11ea-8460-e203acdb5a7d.png)

Ok 'Omitir introducción' is correct and not 'Saltar introdución' or 'Omitir intro' but 'Omitir intro' 

I don't like abbreviations. In the afternoon I change it.

Reply:
you can see the original button text by playing a video from original netflix app

I can do a test run to try to make a button appear with width auto-size
so you could leave the full word
I'll let you know

Reply:
I have just now removed the scrolling when long words are entered here
now the button will adapt according to the text
if you prefer you can also enter full words

Reply:
Thank you @CastagnalT